README.md

```diff
@@ -14,16 +14,9 @@ If you need to try out your changes with consumer applications (e.g. `tagmanager
 - Run `+publishLocal` in sbt (note the `+` makes it cross-compile, e.g. `tagmanager` consumes the 2.11 version)
 - Update the version in the consumer application(s) (e.g. https://github.com/guardian/tagmanager/blob/e47465cbbcdf9e3d3312c5c779eb52fe0676ce4a/build.sbt#L29) using the `-SNAPSHOT` version.
 
-### Publishing a new version
-
-1. Follow the instructions for [publishing a new version to Maven Central via Sonatype](https://docs.google.com/document/d/1rNXjoZDqZMsQblOVXPAIIOMWuwUKe3KzTCttuqS7AcY/edit#).
-   This will include (if not already completed for another project):
-    - Creating and publishing a PGP key
-    - Setting up an account on Sonatype and having it added to the `com.gu` group
-    - Storing your Sonatype credentials in your global sbt configuration
-2. Ensure you're on the branch which holds the changes you're ready to release and that these changes have been approved & tested with the application(s) which use this library (using the `-SNAPSHOT` version).
-3. Ensure the project still builds with `sbt compile`
-4. Run `sbt release`. You will be prompted for a 'release version' – which you should set following semantic versioning as either a patch,
-   minor or major version bump. You will also be prompted for a 'next version' – which should be a patch version ahead of your release version
-   and end `-SNAPSHOT`. `version.sbt` will be updated to reflect this 'next version'.
+## Building a release
 
+This project uses the [`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow)
+to release to Maven Central. To release a new version, execute the
+[Release](https://github.com/guardian/tags-thrift-schema/actions/workflows/release.yml)
+workflow in the Actions tab on GitHub.
```

Comment:
The docs for `gha-scala-library-release-workflow` now have a full screenshots-and-video examples & docs of how to do the release, so you can link to that!


```suggestion
# Publishing a new release

This repo uses [`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow)
to automate publishing releases (both full & preview releases) - see
[**Making a Release**](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/making-a-release.md).
```


_text lifted from https://github.com/guardian/content-api-models?tab=readme-ov-file#publishing-a-new-release_